jctools-experimental/src/main/java/org/jctools/channels/proxy/DemoProxyResult.java

```diff
@@ -48,6 +49,7 @@ public int process(Demo1 impl, int limit) {
             case 2: {
                 float x = UnsafeAccess.UNSAFE.getFloat(rOffset + 4);
                 double y = UnsafeAccess.UNSAFE.getDouble(rOffset + 8);
+                // TODO: Should this offset not be +8 as the last value was a double?
```

Comment:
yes, you are correct, should be:
boolean z = UnsafeAccess.UNSAFE.getBoolean(null, rOffset + 16);